resources/wikia/polyfills/bind.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Polyfill for `Function.prototype.bind` because PhantomJS 1.9.8 doens't support it
+ * Should only need to use this for unit tests, as all Wikia's supported browsers support bind.
+ * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind#Polyfill
+ */
+if (!Function.prototype.bind) {
+	Function.prototype.bind = function(oThis) {
```

Comment:
Makes sense! It would be worse if it was a frequently changed library, but since it's a short polyfill for the sake of tests it makes sense to make it shine like a wikia's diamond :+1: